packages/shop-mobile-ionic/src/app/pages/+products/products.page.ts

```diff
@@ -241,14 +237,17 @@ export class ProductsPage implements OnInit, OnDestroy {
 
 		const count = options['count'];
 		const imageOrientation = options['imageOrientation'];
-
 		this.lastLoadProductsCount = count;
 		this.lastImageOrientation = imageOrientation;
+		const merchants = await this.warehouseService
+			.getAllStores()
+			.toPromise();
+		let merchantIds = [];
 
-		let merchantIds = environment['MERCHANT_IDS'];
```

Comment:
@sunk0 why you removed this? Do you understand why it was here? If not, why you removed???